packages/components/src/image-cropper/styles.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+
+/**
+ * Internal dependencies
+ */
+import ResizableBox from '../resizable-box';
+
+export const PADDING = {
+	x: 30,
+	y: 30,
+};
+
+export const Draggable = styled.div`
+	position: absolute;
+	inset: 0;
+	cursor: move;
+	touch-action: none;
+`;
+
+export const Resizable = styled( ResizableBox )`
+	transform: translate(
+		var( --wp-cropper-window-x ),
+		var( --wp-cropper-window-y )
+	);
+	box-shadow: 0 0 0 100vmax rgba( 0, 0, 0, 0.5 );
```

Comment:
This is a fantastic start. Awesome work @kevin940726 

What do you think about adding some simple guide rulers? I'm using Google photos as inspiration 😄 

https://github.com/user-attachments/assets/1353275d-c429-467d-97e0-193b7404f164



<details>

<summary>Example</summary>

```ts
export const Resizable = styled( ResizableBox )`
	transform: translate(
		var( --wp-cropper-window-x ),
		var( --wp-cropper-window-y )
	);
	box-shadow: 0 0 0 100vmax rgba( 0, 0, 0, 0.6 );
	&:active {
		box-shadow: 0 0 0 100vmax rgba( 0, 0, 0, 0.4 );
		&::after,
		&::before,
		${ Draggable }::after, ${ Draggable }::before {
			position: absolute;
			padding: 0;
			width: 1px;
			height: 100%;
			overflow: hidden;
			left: 33.33%;
			content: ' ';
			background: rgba( 255, 255, 255, 0.33 );
			z-index: 1000;
			display: block;
		}

		&::before {
			right: 33.33%;
			left: auto;
		}

		${ Draggable }::before {
			left: auto;
			width: 100%;
			height: 1px;
			top: 33.33%;
		}

		${ Draggable }::after {
			left: auto;
			width: 100%;
			height: 1px;
			bottom: 33.33%;
			top: auto;
		}
	}
`;

```

</details>


Also, I know it's eye candy, but a subtle animation during the post-crop translate (not dragging) would look pleasing to my eye at least 🤗

Reply:
Nice! I think we can also show it on dragging and rotating too!.

Reply:
I pushed a commit that includes the rulers and the animations!

https://github.com/user-attachments/assets/03a92a8f-9345-488c-89ef-b0e76ebaac34